package.json

```diff
@@ -6,10 +6,8 @@
     "scripts": {
         "anvil": "yarn workspace @river-build/contracts exec anvil",
         "build": "turbo build",
-        "build:react": "turbo build --filter @river-build/react-sdk",
-        "build:metrics-discovery": "turbo build --filter @river-build/metrics-discovery",
         "cast": "yarn workspace @river-build/contracts exec cast",
-        "csb:build": "yarn workspace @river-build/proto run build && yarn workspace @river-build/dlog run build && yarn workspace @river-build/web3 run build && yarn workspace @river-build/encryption run build && yarn workspace @river-build/sdk run build && yarn workspace @river-build/react-sdk run build && yarn workspace stream-metadata run build && echo BUILD DONE || (echo BUILD ERROR; exit 1)",
+        "csb:build": "turbo build --filter @river-build/react-sdk",
```

Comment:
i think you can make this just `@river-build/sdk` 
some history - the "Casablanca" server used to be in typescript, so this command would build the server, client sdk, and shared packages. We no longer have that dependency, so pretty sure all this needs to do is build protobufs and make sure there are no errors on the client side. 

the ci still builds everything right?